github-runners/terraform/base/scripts/secure.sh

```diff
@@ -0,0 +1,249 @@
+#! /bin/bash
```

Comment:
This is a strange file, looks like it was copy-pasted from somewhere and not really applicable for the most part. Why is it necessary?